### ISSUE TITLE
Turn exercise 010 into practice

### ISSUE THREAD

**Issue by cuihtlauac** [ocaml/ocaml.org#2178]
Extract the OCaml code from the markdown file corresponding to this exercise. The file is: `data/exercise/010_NAME.md` (where `010` is the number from the title of this issue and `NAME` is the rest of the file name, for instance `data/exercise/001_tail.md` is the file which corresponds to exercise `001`). Use the code in this file to create a folder `practice/010` (using this same number).

Populate the `010` folder with (almost) the same files as in folders `practice/001` and `practice/002` (they are examples).

Three files are different in each folder:
1. `practice/010/ex.ml`: it contains the function's signature, which needs to be implemented by the learner; 
1. `practice/010/work/impl.ml`: it includes the template meant to be filed by the learner;
1. `practice/010/answer/impl.ml`: consists of the solution from the original markdown file.

The examples included in the original markdown file must be turned into test cases in the `practice/010/ex.ml` file.

Folder structure to create:
```
practice/010/
├── answer
│   ├── impl.ml
│   └── test
│       ├── dune
│       └── run.ml
├── dune
├── dune-project
├── ex.ml
└── work
    ├── impl.ml
    └── test
        ├── dune

**Comment by Kxrishx03** [comment]
Hii! @cuihtlauac Can I take part in it?

**Comment by maha-sachin** [comment]
Hi @cuihtlauac
I’d like to work on this.

**Comment by cuihtlauac** [comment]
@Kxrishx03 pick a single issue to start with

**Comment by rmeis06** [comment]
Hi @cuihtlauac ! Can I work on this issue?

**Comment by cuihtlauac** [comment]
@rmeis06: yes, you're welcome to work on this.

**Comment by cuihtlauac** [comment]
Fixed by #2205
